[Flink AI Flow][Maven] Introduce spotless maven plugin

SteNicholas opened the pull request.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/alibaba/flink-ai-extended?pullRequest=473) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/alibaba/flink-ai-extended?pullRequest=473) before we can accept your contribution.<br/><hr/>**ziyi.jxf** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/alibaba/flink-ai-extended?pullRequest=473) it.</sub>

jiangxin369 commented:
IMO, generated codes(by pb, grpc) should not be formatted

jiangxin369 reviewed (commented):
IMO, generated codes(by pb, grpc) should not be formatted

SteNicholas commented:
@jiangxin369 , thanks for the detailed review. I have addressed your comments and excluded the proto package java format. Please help to review again. Thanks.
